packages/module/src/ChatbotConversationHistoryNav/LoadingState.tsx

```diff
@@ -0,0 +1,39 @@
+import { Skeleton } from '@patternfly/react-core';
+import React from 'react';
+
+export interface LoadingStateProps {
+  'aria-label'?: string;
+}
+export const LoadingState: React.FunctionComponent<LoadingStateProps> = ({
+  'aria-label': ariaLabel
+}: LoadingStateProps) => (
+  <div className="pf-chatbot__history-loading">
+    <div className="pf-chatbot__history-loading-block">
+      <Skeleton screenreaderText={ariaLabel ?? 'Loading'} fontSize="3xl" />
```

Comment:
Rather than just "Loading" as the default, maybe something like "Loading chatbot conversation history"? Could maybe omit "conversation".

Reply:
All set; thanks!